src/Extension/AbstractOrderedExtension.php

```diff
@@ -31,12 +32,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array('position' => null));
+
+        if (method_exists('Symfony\Component\Form\AbstractType', 'configureOptions')) {
```

Comment:
You can check for one of the methods added to the `OptionsResolver` class in 2.6: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/OptionsResolver/CHANGELOG.md